packages/node/package.json

```diff
@@ -45,41 +45,44 @@
     "up": "./scripts/up.py"
   },
   "dependencies": {
-    "@babel/core": "^7.18.5",
+    "@babel/core": "^7.23.9",
     "@bitcoin-computer/lib": "^0.17.2-beta.0",
     "@bitcoin-computer/nakamotojs": "^0.17.2-beta.0",
     "@bitcoin-computer/tiny-secp256k1": "^2.2.1",
-    "@types/chai": "^4.3.1",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "^16.18.16",
-    "@types/sinon": "^10.0.11",
-    "@types/sinon-chai": "^3.2.8",
+    "@types/chai": "^4.3.12",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "^20.11.20",
+    "@types/sinon": "^17.0.3",
+    "@types/sinon-chai": "^3.2.12",
     "argparse": "^2.0.1",
-    "axios-mock-adapter": "^1.21.1",
+    "axios-mock-adapter": "^1.22.0",
     "bitcoind-rpc": "^0.9.1",
-    "body-parser": "^1.19.0",
-    "chai": "^4.3.6",
+    "body-parser": "^1.20.2",
+    "chai": "^5.1.0",
     "cors": "^2.8.5",
-    "dotenv": "^16.0.0",
+    "dotenv": "^16.4.5",
     "ecpair": "^2.1.0",
-    "edit-json-file": "^1.7.0",
+    "edit-json-file": "^1.8.0",
     "elliptic": "^6.5.4",
-    "exponential-backoff": "^3.1.0",
-    "express": "^4.17.1",
-    "express-rate-limit": "^6.2.0",
+    "exponential-backoff": "^3.1.1",
+    "express": "^4.18.2",
+    "express-rate-limit": "^7.1.5",
     "hash.js": "^1.1.7",
-    "mocha": "^10.0.0",
-    "pg-monitor": "^1.4.0",
-    "pg-promise": "^10.11.1",
+    "mocha": "^10.3.0",
+    "pg-monitor": "^2.0.0",
+    "pg-promise": "^11.5.4",
     "run-parallel": "^1.2.0",
-    "sinon": "^14.0.0",
+    "sinon": "^17.0.1",
     "sinon-chai": "^3.7.0",
-    "ts-node": "^10.4.0",
+    "ts-node": "^10.9.2",
     "typeforce": "^1.18.0",
-    "typescript": "^4.9.5",
-    "winston": "^3.3.3",
-    "winston-daily-rotate-file": "^4.7.1",
-    "zeromq": "6.0.0-beta.6"
+    "typescript": "^5.3.3",
+    "winston": "^3.11.0",
+    "winston-daily-rotate-file": "^5.0.0",
+    "zeromq": "6.0.0-beta.19"
+  },
+  "devDependencies": {
+    "@types/bitcoin-computer__lib": "^0.17.2"
```

Comment:
Same here, we should remove this devDependency to see if this change effectively work